core/serialization/blocks.ts

```diff
@@ -709,7 +709,7 @@ function initBlock(block: Block, rendered: boolean) {
       if (isIcon(icon)) {
         icon.onLocationChange(blockSvg.getRelativeToSurfaceXY());
       } else {
-        icon.computeIconLocation();
+        // icon.computeIconLocation();
```

Comment:
Is this change supposed to be here?

Reply:
Whoops, for a second I was gonna do this and #7141 together, but decided to split them out. Looks like this got left behind.